pom.xml

```diff
@@ -2353,7 +2353,7 @@
             <dependency>
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpclient</artifactId>
-                <version>4.5.10</version>
+                <version>4.5.9</version>
```

Comment:
Does this creates the test failures?

Reply:
I confirm. Reverting this solves https://github.com/linagora/james-project/pull/3612#issuecomment-666055406 ...